modules/021-cni-cilium/images/builder/werf.inc.yaml

```diff
@@ -0,0 +1,162 @@
+# https://github.com/cilium/cilium/blob/22161112e06f215a5af9485c05489eba5aa21504/images/runtime/Dockerfile#L8-L11
+---
+artifact: {{ $.ModuleName }}/{{ $.ImageName }}-llvm-artifact
+from: quay.io/cilium/cilium-llvm:3408daa17f6490a464dfc746961e28ae31964c66@sha256:ff13a1a9f973d102c6ac907d2bc38a524c8e1d26c6c1b16ed809a98925206a79
+---
+artifact: {{ $.ModuleName }}/{{ $.ImageName }}-bpftool-artifact
+from: quay.io/cilium/cilium-bpftool:d3093f6aeefef8270306011109be623a7e80ad1b@sha256:2c28c64195dee20ab596d70a59a4597a11058333c6b35a99da32c339dcd7df56
+---
+artifact: {{ $.ModuleName }}/{{ $.ImageName }}-iproute2-artifact
+from: quay.io/cilium/cilium-iproute2:f882e3fd516184703eea5ee9b3b915748b5d4ee8@sha256:f22b8aaf01952cf4b2ec959f0b8f4d242b95ce279480fbd73fded606ce0c3fa4
+---
+artifact: {{ $.ModuleName }}/{{ $.ImageName }}-iptables-artifact
+from: quay.io/cilium/iptables:67f517af50e18f64cd12625021f1c39246bb4f92@sha256:d075f03e89aacf51908346ec8ed5d251b8d3ad528ce30a710fcd074cdf91f11d
+---
+# https://github.com/cilium/cilium/blob/v1.12.8/images/cilium/Dockerfile#L11
+artifact: {{ $.ModuleName }}/{{ $.ImageName }}-envoy-artifact
+from: quay.io/cilium/cilium-envoy:04413917ff99e4f6ab51d1c6eb424d4a055f4462@sha256:af076f80818bc8d894f2f4f3104d5f4288112a67be5fb6e1b9a9c78370c7c9c8
+---
+artifact: {{ $.ModuleName }}/{{ $.ImageName }}-cert-artifact
+from: {{ $.Images.BASE_ALPINE }}
+shell:
+  beforeInstall:
+    - apk add --no-cache ca-certificates
+---
+image: {{ $.ModuleName }}/{{ $.ImageName }}-artifact
+from: {{ $.Images.BASE_GOLANG_19_BULLSEYE }}
+import:
+- artifact: {{ $.ModuleName }}/{{ $.ImageName }}-llvm-artifact
+  add: /usr/local/bin/llvm-objcopy
+  to: /bin
+  before: install
+mount:
+- fromPath: ~/go-pkg-cache
+  to: /go/pkg
+git:
+- add: /modules/021-{{ $.ModuleName }}/images/{{ $.ImageName }}/scripts/builder
+  to: /go/src/github.com/cilium/cilium/images/builder
+  stageDependencies:
+    install:
+    - '**/*'
+shell:
+  beforeInstall:
+  - apt-get update
+  - apt-get upgrade -y --no-install-recommends
+  - |
+    apt-get install -y --no-install-recommends \
+    gcc-aarch64-linux-gnu \
+    g++-aarch64-linux-gnu \
+    libc6-dev-arm64-cross \
+    binutils-aarch64-linux-gnu \
+    gcc-x86-64-linux-gnu \
+    g++-x86-64-linux-gnu \
+    libc6-dev-amd64-cross \
+    binutils-x86-64-linux-gnu \
+    unzip \
+    binutils \
+    coreutils \
+    curl \
+    gcc \
+    git \
+    libc6-dev \
+    patch \
+    make
+  - apt-get purge --auto-remove
+  - apt-get clean
+  - rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+  install:
+  - cd /go/src/github.com/cilium/cilium/images/builder
+  - ./install-protoc.sh
+  - ./install-protoplugins.sh
+---
+artifact: {{ $.ModuleName }}/{{ $.ImageName }}-gops-artifact
+from: {{ $.Images.BASE_GOLANG_19_BULLSEYE }}
+mount:
+  - fromPath: ~/go-pkg-cache
+    to: /go/pkg
+git:
+  - add: /modules/021-{{ $.ModuleName }}/images/{{ $.ImageName }}/scripts/gops
```

Comment:
If we use these scripts without changes, wouldn't it be better to refer the original source instead of storing them in our repo?

Reply:
Good point, big thanks !